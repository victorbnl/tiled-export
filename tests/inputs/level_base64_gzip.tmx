<?xml version="1.0" encoding="UTF-8"?>
<map version="1.9" tiledversion="1.9.2" orientation="orthogonal" renderorder="right-down" width="10" height="5" tilewidth="48" tileheight="48" infinite="0" backgroundcolor="#00aaff" nextlayerid="2" nextobjectid="1">
 <editorsettings>
  <export format="lua"/>
 </editorsettings>
 <tileset firstgid="1" source="tileset.tsx"/>
 <layer id="1" name="Tile Layer 1" width="10" height="5">
  <data encoding="base64" compression="gzip">
   H4sIAAAAAAAAA2NgwA2Y8MhRqo4RitH56JgZimGAGQcGAOPlDubIAAAA
  </data>
 </layer>
</map>
