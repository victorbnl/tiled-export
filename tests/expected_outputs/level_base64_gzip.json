{
  "version": "1.9",
  "tiledversion": "1.9.2",
  "class": "",
  "orientation": "orthogonal",
  "renderorder": "right-down",
  "width": 10,
  "height": 5,
  "tilewidth": 48,
  "tileheight": 48,
  "backgroundcolor": "#00aaff",
  "nextlayerid": 2,
  "nextobjectid": 1,
  "compressionlevel": -1,
  "infinite": false,
  "tilesets": [
    {
      "name": "",
      "firstgid": 1,
      "source": "tileset.tsx"
    }
  ],
  "layers": [
    {
      "x": 0,
      "y": 0,
      "offsetx": 0,
      "offsety": 0,
      "parallaxx": 1,
      "parallaxy": 1,
      "opacity": 1,
      "visible": true,
      "id": 1,
      "name": "Tile Layer 1",
      "class": "",
      "width": 10,
      "height": 5,
      "encoding": "base64",
      "compression": "gzip",
      "data": "H4sIAAAAAAAAA2NgwA2Y8MhRqo4RitH56JgZimGAGQcGAOPlDubIAAAA",
      "type": "tilelayer"
    }
  ],
  "type": "map"
}
